app/email_batching.go

```diff
@@ -140,13 +140,14 @@ func (job *EmailBatchingJob) checkPendingNotifications(now time.Time, handler fu
 		// get how long we need to wait to send notifications to the user
```

Comment:
The code in this file is only executed if batching has been enabled in the System Console, so unlike on the UI, we don't have to check this preference. We can just assume that it is active and use the appropriate default value